test/tictactoe/game_runner_test.clj

```diff
@@ -0,0 +1,106 @@
+(ns tictactoe.game-runner-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [tictactoe.game-runner :as game-runner]))
+
+(defn make-input [coll]
+  (apply str (interleave coll (repeat "\n"))))
+
+(deftest welcome-message-string 
+  (is (= "Welcome to Tic Tac Toe!\n" (game-runner/welcome-message))))
+ 
+(deftest test-game-type-message
+  (is (= "Pick a game type:" (game-runner/game-type-message))))
+
+(deftest test-board-type-message
+  (is (= "What size board would you like to play?" (game-runner/board-type-message))))
+
+(deftest test-pick-player-marker
+  (is (= "Please pick player 1's marker (A-Z)" (game-runner/player-marker-message "player 1"))))
+
+(deftest test-pick-player-two-marker
+  (is (= "Please pick player 2's marker (A-Z)" (game-runner/player-marker-message "player 2"))))
+
+(deftest test-game-choices-message
+  (is (= "You've chosen:" (game-runner/game-choices-message))))
+
+(deftest print-message-test
+  (is (=  "test\n" (with-out-str (game-runner/print-message "test")))))
+
+(deftest get-input-test
+  (is (= "input received" (with-in-str "input received" (game-runner/get-input)))))
+
+(deftest test-loop-and-print
+  (is (= "game => Human vs Human\n" (with-out-str (game-runner/loop-and-print {:game "Human vs Human"})))))
+
+(deftest prints-and-loops
+  (testing "It loops and prints out key value pairs")
+    (with-out-str "test"
+      (is (= nil (game-runner/loop-and-print {:game "Human V Human" :board "3x3"})))))
+
+(deftest loops-when-invalid-marker
+  (testing "When invalid input it loops until valid input is received"
+    (is (= "Invalid. Please pick an available option.\n"
```

Comment:
You might want to check your indentation settings. Generally *most* functions should align arguments like this:

```
(is (= 42
       (+ 21 21)))
```

Certain Clojure functions just indent two spaces without aligning. I would expect `with-*`, `def*`, etc to be that way.